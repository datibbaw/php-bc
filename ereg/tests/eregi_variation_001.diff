006+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
006- Error: 8192 - Function eregi() is deprecated, %s(75)
010+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
010- Error: 8192 - Function eregi() is deprecated, %s(75)
014+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
014- Error: 8192 - Function eregi() is deprecated, %s(75)
018+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
018- Error: 8192 - Function eregi() is deprecated, %s(75)
022+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
022- Error: 8192 - Function eregi() is deprecated, %s(75)
026+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
026- Error: 8192 - Function eregi() is deprecated, %s(75)
030+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
030- Error: 8192 - Function eregi() is deprecated, %s(75)
034+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
034- Error: 8192 - Function eregi() is deprecated, %s(75)
038+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
038- Error: 8192 - Function eregi() is deprecated, %s(75)
043+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
044+ Error: 2 - addcslashes() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(49)
045+ int(1)
046+ Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_001.php(74)
047+ 
048+ Arg value Array 
049+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
050+ Error: 2 - addcslashes() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(49)
051+ int(1)
052+ Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_001.php(74)
053+ 
054+ Arg value Array 
055+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
056+ Error: 2 - addcslashes() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(49)
057+ int(1)
058+ Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_001.php(74)
059+ 
060+ Arg value Array 
061+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
062+ Error: 2 - addcslashes() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(49)
063+ int(1)
043- Error: 8192 - Function eregi() is deprecated, %s(75)
044- Error: 8 - Array to string conversion, %s(75)
045- bool(false)
049- Error: 8192 - Function eregi() is deprecated, %s(75)
050- Error: 8 - Array to string conversion, %s(75)
051- bool(false)
052- Error: 8 - Array to string conversion, %seregi_variation_001.php(%d)
054- Arg value Array 
055- Error: 8192 - Function eregi() is deprecated, %s(75)
056- Error: 8 - Array to string conversion, %s(75)
057- bool(false)
067+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
068+ Error: 2 - addcslashes() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(49)
069+ int(1)
070+ 
071+ Arg value  
072+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
073+ int(1)
074+ 
075+ Arg value  
076+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
077+ int(1)
078+ 
079+ Arg value 1 
080+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
081+ int(1)
082+ 
083+ Arg value  
084+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
085+ int(1)
086+ 
087+ Arg value 1 
088+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
089+ int(1)
090+ 
091+ Arg value  
092+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
093+ int(1)
094+ 
095+ Arg value  
096+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
097+ int(1)
098+ 
099+ Arg value  
100+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
101+ int(1)
102+ Error: 4096 - Object of class stdClass could not be converted to string, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_001.php(74)
103+ 
104+ Arg value  
105+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
106+ Error: 4096 - Object of class stdClass could not be converted to string, /Users/tjerk/Documents/php-bc/ereg/ereg.php(46)
107+ int(1)
108+ 
109+ Arg value  
110+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
111+ int(1)
113+ Arg value  
114+ Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
115+ int(1)
116+ Done
058- Error: 8 - Array to string conversion, %seregi_variation_001.php(%d)
059- 
060- Arg value Array 
061- Error: 8192 - Function eregi() is deprecated, %s(75)
062- Error: 8 - Array to string conversion, %s(75)
063- bool(false)
064- Error: 8 - Array to string conversion, %seregi_variation_001.php(%d)
065- 
066- Arg value Array 
067- Error: 8192 - Function eregi() is deprecated, %s(75)
068- Error: 8 - Array to string conversion, %s(75)
069- bool(false)
070- 
071- Arg value  
072- Error: 8192 - Function eregi() is deprecated, %s(75)
073- Error: 2 - eregi(): REG_EMPTY, %s(75)
074- bool(false)
075- 
076- Arg value  
077- Error: 8192 - Function eregi() is deprecated, %s(75)
078- Error: 2 - eregi(): REG_EMPTY, %s(75)
079- bool(false)
080- 
081- Arg value 1 
082- Error: 8192 - Function eregi() is deprecated, %s(75)
083- int(1)
084- 
085- Arg value  
086- Error: 8192 - Function eregi() is deprecated, %s(75)
087- Error: 2 - eregi(): REG_EMPTY, %s(75)
088- bool(false)
089- 
090- Arg value 1 
091- Error: 8192 - Function eregi() is deprecated, %s(75)
092- int(1)
093- 
094- Arg value  
095- Error: 8192 - Function eregi() is deprecated, %s(75)
096- Error: 2 - eregi(): REG_EMPTY, %s(75)
097- bool(false)
098- 
099- Arg value  
100- Error: 8192 - Function eregi() is deprecated, %s(75)
101- Error: 2 - eregi(): REG_EMPTY, %s(75)
102- bool(false)
103- 
104- Arg value  
105- Error: 8192 - Function eregi() is deprecated, %s(75)
106- Error: 2 - eregi(): REG_EMPTY, %s(75)
107- bool(false)
108- Error: 4096 - Object of class stdClass could not be converted to string, %s(74)
109- 
110- Arg value  
111- Error: 8192 - Function eregi() is deprecated, %s(75)
112- Error: 4096 - Object of class stdClass could not be converted to string, %s(75)
113- Error: 8 - Object of class stdClass to string conversion, %s(75)
114- bool(false)
115- 
116- Arg value  
117- Error: 8192 - Function eregi() is deprecated, %s(75)
118- Error: 2 - eregi(): REG_EMPTY, %s(75)
119- bool(false)
120- 
121- Arg value  
122- Error: 8192 - Function eregi() is deprecated, %s(75)
123- Error: 2 - eregi(): REG_EMPTY, %s(75)
124- bool(false)
125- Done