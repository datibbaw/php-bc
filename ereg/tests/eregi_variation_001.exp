*** Testing eregi() : usage variations ***
Error: 8 - Undefined variable: undefined_var, %s(65)
Error: 8 - Undefined variable: unset_var, %s(68)

Arg value 0 
Error: 8192 - Function eregi() is deprecated, %s(75)
bool(false)

Arg value 1 
Error: 8192 - Function eregi() is deprecated, %s(75)
int(1)

Arg value 12345 
Error: 8192 - Function eregi() is deprecated, %s(75)
bool(false)

Arg value -2345 
Error: 8192 - Function eregi() is deprecated, %s(75)
bool(false)

Arg value 10.5 
Error: 8192 - Function eregi() is deprecated, %s(75)
bool(false)

Arg value -10.5 
Error: 8192 - Function eregi() is deprecated, %s(75)
bool(false)

Arg value 101234567000 
Error: 8192 - Function eregi() is deprecated, %s(75)
bool(false)

Arg value 1.07654321E-9 
Error: 8192 - Function eregi() is deprecated, %s(75)
bool(false)

Arg value 0.5 
Error: 8192 - Function eregi() is deprecated, %s(75)
bool(false)
Error: 8 - Array to string conversion, %seregi_variation_001.php(%d)

Arg value Array 
Error: 8192 - Function eregi() is deprecated, %s(75)
Error: 8 - Array to string conversion, %s(75)
bool(false)
Error: 8 - Array to string conversion, %seregi_variation_001.php(%d)

Arg value Array 
Error: 8192 - Function eregi() is deprecated, %s(75)
Error: 8 - Array to string conversion, %s(75)
bool(false)
Error: 8 - Array to string conversion, %seregi_variation_001.php(%d)

Arg value Array 
Error: 8192 - Function eregi() is deprecated, %s(75)
Error: 8 - Array to string conversion, %s(75)
bool(false)
Error: 8 - Array to string conversion, %seregi_variation_001.php(%d)

Arg value Array 
Error: 8192 - Function eregi() is deprecated, %s(75)
Error: 8 - Array to string conversion, %s(75)
bool(false)
Error: 8 - Array to string conversion, %seregi_variation_001.php(%d)

Arg value Array 
Error: 8192 - Function eregi() is deprecated, %s(75)
Error: 8 - Array to string conversion, %s(75)
bool(false)

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(75)
Error: 2 - eregi(): REG_EMPTY, %s(75)
bool(false)

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(75)
Error: 2 - eregi(): REG_EMPTY, %s(75)
bool(false)

Arg value 1 
Error: 8192 - Function eregi() is deprecated, %s(75)
int(1)

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(75)
Error: 2 - eregi(): REG_EMPTY, %s(75)
bool(false)

Arg value 1 
Error: 8192 - Function eregi() is deprecated, %s(75)
int(1)

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(75)
Error: 2 - eregi(): REG_EMPTY, %s(75)
bool(false)

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(75)
Error: 2 - eregi(): REG_EMPTY, %s(75)
bool(false)

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(75)
Error: 2 - eregi(): REG_EMPTY, %s(75)
bool(false)
Error: 4096 - Object of class stdClass could not be converted to string, %s(74)

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(75)
Error: 4096 - Object of class stdClass could not be converted to string, %s(75)
Error: 8 - Object of class stdClass to string conversion, %s(75)
bool(false)

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(75)
Error: 2 - eregi(): REG_EMPTY, %s(75)
bool(false)

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(75)
Error: 2 - eregi(): REG_EMPTY, %s(75)
bool(false)
Done