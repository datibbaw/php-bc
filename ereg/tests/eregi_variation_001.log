
---- EXPECTED OUTPUT
*** Testing eregi() : usage variations ***
Error: 8 - Undefined variable: undefined_var, %s(65)
Error: 8 - Undefined variable: unset_var, %s(68)

Arg value 0 
Error: 8192 - Function eregi() is deprecated, %s(75)
bool(false)

Arg value 1 
Error: 8192 - Function eregi() is deprecated, %s(75)
int(1)

Arg value 12345 
Error: 8192 - Function eregi() is deprecated, %s(75)
bool(false)

Arg value -2345 
Error: 8192 - Function eregi() is deprecated, %s(75)
bool(false)

Arg value 10.5 
Error: 8192 - Function eregi() is deprecated, %s(75)
bool(false)

Arg value -10.5 
Error: 8192 - Function eregi() is deprecated, %s(75)
bool(false)

Arg value 101234567000 
Error: 8192 - Function eregi() is deprecated, %s(75)
bool(false)

Arg value 1.07654321E-9 
Error: 8192 - Function eregi() is deprecated, %s(75)
bool(false)

Arg value 0.5 
Error: 8192 - Function eregi() is deprecated, %s(75)
bool(false)
Error: 8 - Array to string conversion, %seregi_variation_001.php(%d)

Arg value Array 
Error: 8192 - Function eregi() is deprecated, %s(75)
Error: 8 - Array to string conversion, %s(75)
bool(false)
Error: 8 - Array to string conversion, %seregi_variation_001.php(%d)

Arg value Array 
Error: 8192 - Function eregi() is deprecated, %s(75)
Error: 8 - Array to string conversion, %s(75)
bool(false)
Error: 8 - Array to string conversion, %seregi_variation_001.php(%d)

Arg value Array 
Error: 8192 - Function eregi() is deprecated, %s(75)
Error: 8 - Array to string conversion, %s(75)
bool(false)
Error: 8 - Array to string conversion, %seregi_variation_001.php(%d)

Arg value Array 
Error: 8192 - Function eregi() is deprecated, %s(75)
Error: 8 - Array to string conversion, %s(75)
bool(false)
Error: 8 - Array to string conversion, %seregi_variation_001.php(%d)

Arg value Array 
Error: 8192 - Function eregi() is deprecated, %s(75)
Error: 8 - Array to string conversion, %s(75)
bool(false)

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(75)
Error: 2 - eregi(): REG_EMPTY, %s(75)
bool(false)

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(75)
Error: 2 - eregi(): REG_EMPTY, %s(75)
bool(false)

Arg value 1 
Error: 8192 - Function eregi() is deprecated, %s(75)
int(1)

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(75)
Error: 2 - eregi(): REG_EMPTY, %s(75)
bool(false)

Arg value 1 
Error: 8192 - Function eregi() is deprecated, %s(75)
int(1)

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(75)
Error: 2 - eregi(): REG_EMPTY, %s(75)
bool(false)

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(75)
Error: 2 - eregi(): REG_EMPTY, %s(75)
bool(false)

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(75)
Error: 2 - eregi(): REG_EMPTY, %s(75)
bool(false)
Error: 4096 - Object of class stdClass could not be converted to string, %s(74)

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(75)
Error: 4096 - Object of class stdClass could not be converted to string, %s(75)
Error: 8 - Object of class stdClass to string conversion, %s(75)
bool(false)

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(75)
Error: 2 - eregi(): REG_EMPTY, %s(75)
bool(false)

Arg value  
Error: 8192 - Function eregi() is deprecated, %s(75)
Error: 2 - eregi(): REG_EMPTY, %s(75)
bool(false)
Done
---- ACTUAL OUTPUT
*** Testing eregi() : usage variations ***
Error: 8 - Undefined variable: undefined_var, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_001.php(65)
Error: 8 - Undefined variable: unset_var, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_001.php(68)

Arg value 0 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)

Arg value 1 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value 12345 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)

Arg value -2345 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)

Arg value 10.5 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)

Arg value -10.5 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)

Arg value 101234567000 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)

Arg value 1.07654321E-9 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)

Arg value 0.5 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_001.php(74)

Arg value Array 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
Error: 2 - addcslashes() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(49)
int(1)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_001.php(74)

Arg value Array 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
Error: 2 - addcslashes() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(49)
int(1)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_001.php(74)

Arg value Array 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
Error: 2 - addcslashes() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(49)
int(1)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_001.php(74)

Arg value Array 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
Error: 2 - addcslashes() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(49)
int(1)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_001.php(74)

Arg value Array 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
Error: 2 - addcslashes() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(49)
int(1)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value 1 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value 1 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)
Error: 4096 - Object of class stdClass could not be converted to string, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_001.php(74)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
Error: 4096 - Object of class stdClass could not be converted to string, /Users/tjerk/Documents/php-bc/ereg/ereg.php(46)
int(1)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)
Done
---- FAILED
