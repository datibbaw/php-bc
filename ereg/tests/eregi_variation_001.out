*** Testing eregi() : usage variations ***
Error: 8 - Undefined variable: undefined_var, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_001.php(65)
Error: 8 - Undefined variable: unset_var, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_001.php(68)

Arg value 0 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)

Arg value 1 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value 12345 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)

Arg value -2345 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)

Arg value 10.5 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)

Arg value -10.5 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)

Arg value 101234567000 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)

Arg value 1.07654321E-9 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)

Arg value 0.5 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
bool(false)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_001.php(74)

Arg value Array 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
Error: 2 - addcslashes() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(49)
int(1)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_001.php(74)

Arg value Array 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
Error: 2 - addcslashes() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(49)
int(1)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_001.php(74)

Arg value Array 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
Error: 2 - addcslashes() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(49)
int(1)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_001.php(74)

Arg value Array 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
Error: 2 - addcslashes() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(49)
int(1)
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_001.php(74)

Arg value Array 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
Error: 2 - addcslashes() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(49)
int(1)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value 1 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value 1 
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)
Error: 4096 - Object of class stdClass could not be converted to string, /Users/tjerk/Documents/php-bc/ereg/tests/eregi_variation_001.php(74)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
Error: 4096 - Object of class stdClass could not be converted to string, /Users/tjerk/Documents/php-bc/ereg/ereg.php(46)
int(1)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)

Arg value  
Error: 16384 - Function eregi() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(108)
int(1)
Done